collector/os.go

```diff
@@ -30,77 +30,66 @@ type OSCollector struct {
 
 // NewOSCollector ...
 func NewOSCollector() (Collector, error) {
-
 	const subsystem = "os"
 
 	return &OSCollector{
-
 		PagingMaxBytes: prometheus.NewDesc(
 			prometheus.BuildFQName(Namespace, subsystem, "paging_max_bytes"),
-			"SizeStoredInPagingFiles",
+			"OperatingSystem.SizeStoredInPagingFiles",
 			nil,
 			nil,
 		),
-
 		PagingFreeBytes: prometheus.NewDesc(
 			prometheus.BuildFQName(Namespace, subsystem, "paging_free_bytes"),
-			"FreeSpaceInPagingFiles",
+			"OperatingSystem.FreeSpaceInPagingFiles",
 			nil,
 			nil,
 		),
-
 		PhysicalMemoryFreeBytes: prometheus.NewDesc(
 			prometheus.BuildFQName(Namespace, subsystem, "physical_memory_free_bytes"),
-			"FreePhysicalMemory",
+			"OperatingSystem.FreePhysicalMemory",
 			nil,
 			nil,
 		),
-
 		Processes: prometheus.NewDesc(
 			prometheus.BuildFQName(Namespace, subsystem, "processes"),
-			"NumberOfProcesses",
+			"OperatingSystem.NumberOfProcesses",
 			nil,
 			nil,
 		),
-
 		ProcessesMax: prometheus.NewDesc(
 			prometheus.BuildFQName(Namespace, subsystem, "processes_max"),
-			"MaxNumberOfProcesses",
+			"OperatingSystem.MaxNumberOfProcesses",
```

Comment:
Shouldn't we ditch this one? We've got `NumberOfProcesses` already, so we can use PromQL to get the max